packages/Php70/tests/Rector/FuncCall/NonVariableToVariableOnFunctionCallRector/Fixture/method_calls.php.inc

```diff
@@ -65,6 +68,10 @@ function methodCalls()
     $bar = bar();
     $aClass->child()->bar($bar);
 
+    $methodName = AClass::class.'::staticMethod';
+    $bar = bar();
+    $methodName($bar);
+
     $anonymousClass = new class {
```

Comment:
It would be better to decouple those into own test case. It starts to be pretty huge and then very costly to debug later

Reply:
Done.